### ISSUE TITLE
How to create Ground Truth and Prediction txt files

### ISSUE THREAD

**Issue by hiteshkhanna31** [Cartucho/mAP#113]
I am using Tensor flow 2 for YOLOV3 (https://github.com/pythonlessons/TensorFlow-2.x-YOLOv3) and i want to evaluate it on MS COCO validation dataset. How should the prediction code be modifed so that i have a detection txt generate for every image and how can i make Groundtruth txt from instance_val2017.json files available in the dataset.

I want to use this Object detection metrics to calculate mAP.

Appreciate if someone can help as i am beginner in Computer vision domain.

**Comment by takehiro-code** [comment]
In my understanding, this repository computes mAP (also called as AP) based on _PASCAL VOC2012 challenge_. The metric mAP is computed based on the average of all object classes and IOU threshold at 0.5. However, if you want to evaluate based on _COCO challenge_, mAP will be computed based on the average of all object classes AND average of 10 IOU thresholds of 0.5 to 0.95. I think modification of mAP definition to COCO challenge may be necessary, which I hope as a new feature or enhancement.

**Comment by Ash-Chump2Champ** [comment]
Hi so I am using that script to convert YOLO to our format:
I would like to ask, do I need to use the .cfg file for training or testing?
# Testing
batch=1
subdivisions=1
# Training
#batch=64
#subdivisions=16
